src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -63,6 +63,7 @@
 public class Checker extends AbstractAutomaticBean implements MessageDispatcher, RootModule {
 
     /** Message to use when an exception occurs and should be printed as a violation. */
+    @MessageKey
     public static final String EXCEPTION_MSG = "general.exception";
```

Comment:
How does this make things easier when it isn't even a check? Issue doesn't define, but if this for all classes that can be instantiated by `PackageObjectFactory`?

Reply:
@romani Please clarify what should be hit by this. The issue should be updated with this.

Are we only talking Checks? Anything that can come from `PackageObjectFactory`? AuditListeners technically have messages, are they included?

Reply:
I updated issue title and description

Reply:
Done. Removed.